Add warning about keeping Single instance initialization as the first one (#3334)
tauri-apps/tauri-docs: src/content/docs/plugin/single-instance.mdx

</Tabs>

> 

Reviewer: ```suggestion
```

:::info

> The Single Instance plugin must be the first one to be init()'ed to work well. Doing so prevents other plugins to initiate and eventually interfering with the app's instance that is already running.

Reviewer: ```suggestion
The Single Instance plugin must be the first one to be registered to work well. This assures that it runs before other plugins can interfere.
```
